src/components/SearchUser/index.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './style.css';
+const SearchUser = () => {
+    
+    const handleFormSubmit = (e) => {
+        e.preventDefault()
+        localStorage.setItem('username', e.target[0].value);
```

Comment:
Nice using Local storage to keep the username.
Would be nice to have a signout button to clear it